### ISSUE TITLE
Heatmap Series deep dive debugging 

### ISSUE THREAD

**Issue by cdolfi** [oss-aspen/8Knot#696]

**Comment by cdolfi** [comment]
https://github.blog/changelog/2022-06-06-view-commit-history-across-file-renames-and-moves/

**Comment by EngCaioFonseca** [comment]
@cdolfi still relevant?

**Comment by cdolfi** [comment]
@EngCaioFonseca yes, I do think this should be reviewed and most likely improved on from its initial implementation
